glossary.yml

```
@@ -1919,7 +1955,26 @@
       the relationships between values, and ways to access or manipulate those data, such as
       reading, altering, or writing.
 
+- slug: data_type
+  en:
+    term: "data type"
+    def: >
+      An attribute of data which defines what operations can be performed
+      and defines the range of values that can be stored in a data object
+      of a given type.
+      Data types include: boolean or logical, binary, integer, 
+      floating point numeric, character or string.     
```

```suggestion
      An attribute of data which defines what operations can be performed
      and defines the range of values that can be stored in a data object
      of a given type. Common data types include: boolean or logical, 
      binary, integer, floating point numeric, character or string.     
```

This needs the additional caveat because there are many other data types: enums, pointers, abstract data types (like lists, bags, and sets), et cetera, and even ones that are more specialised and language-dependent, like None, void, or unit.

---

Yes that was the intended meaning. I think some examples help but listing them all would be cumbersome. 😄 

```suggestion
      An attribute of data which defines what operations can be performed
      and defines the range of values that can be stored in a data object
      of a given type. Examples of common types include: boolean or logical,
      binary, integer,  floating point numeric, character or string.     
```